config.py

```diff
@@ -121,16 +132,60 @@ def fromParser(self, name, parser):
         try:
             rhsm_password = parser.get(name, "rhsm_password")
         except NoOptionError:
+            rhsm_password = None
+
+        try:
+            rhsm_host = parser.get(name, "rhsm_host")
```

Comment:
So we're largely re-creating the [server] section of rhsm.conf here, with an rhsm_ prefix. As such we should probably keep property names exactly the same, so could we make this one rhsm_hostname.